stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/database_logic.py

```diff
@@ -1933,8 +1928,6 @@ async def execute_discovery_search(
 
         query = search.query.to_dict() if search.query else None
 
-        index_param = "document"  # indices(collection_ids)
```

Comment:
document isn't needed anymore?

Reply:
Yes, I believe this was included as a placeholder, but we can instead directly code the index into the function calls on line #1944 and #1237 respectively.